smg-automotive/components-pkg
# feat: add flex and heading

[VSST-507](https://autoricardo.atlassian.net/browse/VSST-507?atlOrigin=eyJpIjoiNWViMDFjYTdhNTA5NDBkMWJlYjE1OGIzZDZkM2E5MDUiLCJwIjoiaiJ9) and [VSST-508](https://autoricardo.atlassian.net/browse/VSST-508?atlOrigin=eyJpIjoiZGRmNmE4N2ViM2M2NDVhZTk0OTNiMmI5NWFhYjBkZWMiLCJwIjoiaiJ9)

## Motivation and context

Extract Flex and heading components

## Before

We had to import Chakra in our projects to use Flex or heading components

## After

Flex and Heading are now parts of component-pkg

## How to test

Please check storybook and [this PR](https://github.com/smg-automotive/seller-web/pull/7).

Thank you 🦞
